### DISCUSSION TITLE
V0.0.8

### DISCUSSION THREAD

**Discussion by MFn233** [MFn233/PCL-Mainpage#23] [公告]
首先，我将冗余的TextBlock元素合并到了HintText属性中，如：
![image](https://github.com/MFn233/PCL-Mainpage/assets/103323756/1a8bbd44-5fb2-499d-9da1-2dff9bc1fc49)
![image](https://github.com/MFn233/PCL-Mainpage/assets/103323756/f7e90d94-ee6d-4184-a3e7-0d14d957ae47)
*感谢@Ad-closeNN，@SYBass*

其次，我删除了MCBBS，增加了停止运行的知乎专栏（知乎chenskiro）
![image](https://github.com/MFn233/PCL-Mainpage/assets/103323756/3b538993-5d2e-45c3-a666-e91f9bc587c3)
![image](https://github.com/MFn233/PCL-Mainpage/assets/103323756/064c8082-0e09-4b84-8770-5955f3b22600)

我将“杂项”改为“更多”，并增加了新项目“主页市场”
![image](https://github.com/MFn233/PCL-Mainpage/assets/103323756/c8e3cbaa-7262-4f9b-97fc-bd3276002b1d)
![image](https://github.com/MFn233/PCL-Mainpage/assets/103323756/ac4c949f-28c8-4682-bff3-0f8fc4cc91de)

同时，我与其他人对部分地方进行了微调，详见完整更新日志。

## What's Changed
* 一堆修改 by @MCSteve123 in https://github.com/MFn233/PCL-Mainpage/pull/16
* 继MCSteve 123后的优化 by @Ad-closeNN in https://github.com/MFn233/PCL-Mainpage/pull/20

## New Contributors
* @MCSteve123 made their first contribution in https://github.com/MFn233/PCL-Mainpage/pull/16

**Full Changelog**: https://github.com/MFn233/PCL-Mainpage/compare/V0.0.7...V0.0.8

<hr /><em>This discussion was created from the release <a href='https://github.com/MFn233/PCL-Mainpage/releases/tag/V0.0.8'>V0.0.8</a>.</em>

**Comment by MFn233** [comment]
肝！我的肝！

**Reply by Ad-closeNN** [reply]
~~名字！我的名字~~

**Comment by MFn233** [comment]
这也是一次悲伤的更新

**Reply by MFn233** [reply]
SIPC跟我说他没法同步到cdn上，因为域名
![image](https://github.com/MFn233/PCL-Mainpage/assets/103323756/3643965a-9409-43ff-97bc-8f36ed7e7c26)

**Comment by SYBass** [comment]
没看到域名啊

**Reply by MFn233** [reply]
你没看到图片吗：）

**Reply by MFn233** [reply]
不过他还是同步了，很神奇

**Reply by SYBass** [reply]
看着了

**Reply by SYBass** [reply]
枳领主页为啥会报错？
